.github/workflows/build.yml

```diff
@@ -110,18 +110,13 @@ jobs:
       with:
         platforms: all
 
-    - name: Print build identifiers
-      run: |
-        python -m pip install cibuildwheel==2.10.1
-        python -m cibuildwheel --only ${{ matrix.cibw-only }} --print-build-identifiers
-
```

Comment:
What was this for? It's just printing out exactly the `--only`?

Reply:
It's just making sure that the arguments are being passed through to cibuildwheel correctly, otherwise you have to make a tagged release to invoke cibuildwheel for this combination. For that reason, I'd say keep it in.